ddtrace/internal/dogstatsd.py

```diff
@@ -1,12 +1,13 @@
+from typing import List
 from typing import Optional
 
 from ddtrace.internal.compat import parse
 from ddtrace.vendor.dogstatsd import DogStatsd
 from ddtrace.vendor.dogstatsd import base
 
 
-def get_dogstatsd_client(url, namespace=None):
-    # type: (str, Optional[str]) -> DogStatsd
+def get_dogstatsd_client(url, namespace=None, tags=None):
```

Comment:
Is this change related to this integration?

Reply:
pulled out into a separate change #5564